exercises/luhn-trait/src/lib.rs

```diff
@@ -1 +1,14 @@
+pub trait Luhn {
+    fn valid_luhn(&self) -> bool;
+}
 
+/// Here is the example of how to implement custom Luhn trait
+/// for the &str type. Naturally, you can implement this trait
+/// by hand for the every other type presented in the test suite,
+/// but your solution will fail if a new type is presented.
+/// Perhaps there exists a better solution for this problem?
+impl Luhn for &'static str {
```

Comment:
Same comment as in `luhn-from`: if we claim we're going to implement this for `&str`, then we should do it properly, with `impl<'a> Luhn for &'a str { ... }`.